lib/pbench/agent/tool_meister_start.py

```diff
@@ -1358,15 +1358,19 @@ def main():
         dest="redis_server",
         default=os.environ.get("PBENCH_REDIS_SERVER", None),
         help=(
-            "Specifies the IP/port to use for the Redis server - if not"
-            " present, the defaults are used, ${_pbench_full_hostname}:"
+            "Specifies the IP/port to use for the Redis server - if not present"
+            " (and if not supplied via the PBENCH_REDIS_SERVER environment"
+            " variable), the defaults are used, ${_pbench_full_hostname}:"
             f"{def_redis_port};"
             " the specified value can take either of two forms:"
             " `<bind host>:<port>;<host>:<port>`, a semi-colon separated"
             " IP/port specified for both how the Redis server will bind"
             " itself, and how clients will connect; `<host>:<port>`, the"
             " IP/port combination is used both for binding and connecting"
             " (NOTE: binding is not used with --orchestrate=existing);"
+            " NOTE: if this option is specified, it is assumed the caller"
+            " has created and provisioned the specified Redis server, and"
```

Comment:
Two _NOTE_ phrases in a row, with different formatting, looks odd. Either parenthesize both or neither? Perhaps they should be separate *sentences*? This is a really long "sentence" strung with mixed phrase punctuation that I think is not particularly helpful. ☹️

Reply:
OK, I broke it up into a sequence of sentences.  And, of course, I had to do the next switch, too....